docset/winserver2012r2-ps/hpc/Get-HpcTestDetail.md

```diff
@@ -146,8 +146,16 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### HpcTestCaseDetail, HpcTestCase<Type>Parameter, HpcVariable, HpcTestCaseStep
-This cmdlet returns an **HpcTestCaseDetail** object, zero or more **HpcTestCase\<Type\>Parameter** objects, zero or more **HpcVariable** objects, and one to three **HpcTestCaseStep** objects.
+### HpcTestCaseDetail
+
+### HpcTestCase
```

Comment:
Not sure about this one. `HpcTestCase<Type>Parameter` appears incorrect, but wasn't sure if it should be a list type

Reply:
hmm, not sure too, I will have to track down a product team dev to find out

Reply:
I have a project to use the cmdlet updater tool I use for Office PowerShell and will use it over here to audit these..  here is tool just FYI.. it is still being iterated on but useful already.. it uses PlatyPS open source tool but streamlines submitting PR's. https://github.com/MicrosoftDocs/office-docs-powershell/tree/master/tools/office-cmdlet-updater